app/javascript/mastodon/components/icon_button.js

```diff
@@ -83,8 +83,8 @@ export default class IconButton extends React.PureComponent {
   render () {
     const style = {
       fontSize: `${this.props.size}px`,
-      width: `${this.props.size * 1.28571429}px`,
-      height: `${this.props.size * 1.28571429}px`,
+      width: '1.28571429em',
+      height: '1.28571429em',
```

Comment:
Is there any reason for the very specific w/h? Would `1.285` not work?

Reply:
I don't know, I'm just using the previously-used value.